lang/java/reef-io/src/main/java/org/apache/reef/io/network/impl/NetworkService.java

```diff
@@ -81,63 +61,17 @@
   private Identifier myId;
 
   /**
-   * @param factory
-   * @param nsPort
-   * @param nameServerAddr
-   * @param nameServerPort
-   * @param codec
-   * @param tpFactory
-   * @param recvHandler
-   * @param exHandler
-   * @deprecated have an instance injected instead.
-   */
-  @Deprecated
-  public NetworkService(final IdentifierFactory factory,
```

Comment:
Same as on the other PR: Please do not remove these old constructors. Instead, be super-explicit in the `@deprecated` message and file a JIRA conditioned on release 0.12 to delete these.